src/satellite/test.js

```diff
@@ -102,12 +102,11 @@ describe('Satellite()', () => {
       service.start(port, async () => {
         const res = await fetch(`${url}/healthcheck`);
         expect(res.ok).toBe(false);
-        await service.stop();
-        done();
+        return service.stop();
```

Comment:
I understand now.  So you should deal with stopping the service in `afterEach` vs here, and go back to your `done()` approach.

In all tests, you are basically doing:

```js
let service;

beforeEach(() => {
  service = createService();
  ...
});

afterEach(() => service.stop())

test('...', async (done) => {
  service.start(..., () => {
    ...
    done();
  });
});
```

Reply:
I don't think function can be `async` and take `done`

Reply:
Why do you say that?

Reply:
Actually, it's just test function, Jest just told me this 
```
    Test functions cannot both take a 'done' callback and return something. Either use a 'done' callback, or return a promise.

```

Reply:
Move the `async` off of the outer callback function to test:

```js
test('...', (done) => {
  service.start(..., async () => {
    await whatever();
    done;
  });
});
```